main.webapp/build.gradle

```diff
@@ -4,6 +4,10 @@ apply plugin: 'com.moowork.grunt'
 
 mainClassName = "org.starchartlabs.river.main.webapp.River"
 
```

Comment:
This buys us some built-in tasks and conventions, but is more meant for Java sources and such - on the fence about whether using sourceSets like this is an abuse